include/motor-control/core/motion_controller.hpp

```diff
@@ -50,13 +50,17 @@ class MotionController {
                  .velocity = velocity_steps,
                  .acceleration = acceleration_steps,
                  .group_id = can_msg.group_id,
-                 .seq_id = can_msg.seq_id};
+                 .seq_id = can_msg.seq_id,
+                 .stop_condition = static_cast<MoveStopCondition>(
```

Comment:
Think you can just pass this into the enum like: `MoveStopCondition(can_msg.request_stop_condition)`